mummer/4.0.0/Dockerfile

```diff
@@ -0,0 +1,75 @@
+# FROM defines the base docker image. This command has to come first in the file
+# The 'as' keyword lets you name the following stage. We use `app` for the production image
+FROM ubuntu:xenial as app
+
+# ARG sets environment variables during the build stage
+ARG MUMMER_VER="4.0.0"
+
+# LABEL instructions tag the image with metadata that might be important to the user
+LABEL base.image="ubuntu:xenial"
+LABEL dockerfile.version="1"
+LABEL software="mummer"
+LABEL software.version=$MUMMER_VER
+LABEL description="MUMmer is a versatile alignment tool for DNA and protein sequences."
+LABEL website="https://github.com/StaPH-B/docker-builds"
+LABEL license="https://github.com/StaPH-B/docker-builds/blob/master/LICENSE"
+LABEL maintainer="John Arnn"
+LABEL maintainer.email="jarnn@utah.gov"
+
+# RUN executes code during the build
+# Install dependencies via apt-get or yum if using a centos or fedora base
+RUN apt-get update && apt-get install -y \
+ wget \
+ git \
+ libncurses5-dev \
+ libbz2-dev \
+ liblzma-dev \
+ libcurl4-gnutls-dev \
+ zlib1g-dev \
+ libssl-dev \
+ gcc \
+ make \
+ perl \
+ bzip2 \
+ gnuplot \
+ ca-certificates \
+ gawk \
+ curl \
+ sed \
+ sudo \
```

Comment:
I would recommend removing `sudo` here. It is not necessary since when the docker image is built, it is by default built as the `root` user. 

Additionally you can remove `sudo` from line 51